frontend/src/static/js/v_authorship.js

```diff
@@ -16,6 +16,7 @@ window.vAuthorship = {
     return {
       isLoaded: false,
```

Comment:
This attribute should be also removed right?

Reply:
Actually I realized that without this, `nothing to see here :(` might be wrongly displayed due to async loading of data. Unless having it displayed for a few seconds while data is loading doesn't matter then I suppose it is fine to remove it.

Reply:
Is it because you forgot to remove the `v-if` here:
![image](https://user-images.githubusercontent.com/35621726/102688915-29fbcd80-4235-11eb-8043-b3056edf33a2.png)

If what you mention is because moving this `if`, to fix that, replace it with a mapped computed value from vue store.

Reply:
![image](https://user-images.githubusercontent.com/16630400/102710255-0a73ac00-42ec-11eb-9132-aa41133707dd.png)

This is the async function which is used to load the files (from JSON). Because of how Vue works the browser will be rendered before this function ends. Since the `files` array depends on waiting for this function there isn't any computed value from vue store to map from until the function finishes.

Reply:
OK, then, we can just keep this attribute